### ISSUE TITLE
autofix: python objects gets reformatting wrongly in yaml files

### ISSUE THREAD

**Issue by ssbarnea** [ansible/ansible-lint#4222]
##### Summary

When using autofix a change like below will be seen, which breaks the files. This was observed with devtools mkdocs.yml files, which happen to make use of this unpopular YAML feature:

```
format: !!python/name:pymdownx.superfences.fence_code_format
format: !!%20%20python/name:pymdownx.superfences.fence_code_format
```

##### Issue Type

- Bug Report

##### OS / ENVIRONMENT

```console (paste below)
ansible-lint --version
```

- ansible installation method: one of source, pip, OS package
- ansible-lint installation method: one of source, pip, OS package

##### STEPS TO REPRODUCE

```console (paste below)

```

##### Desired Behavior

Possible security bugs should be reported via email to `security@ansible.com`

##### Actual Behavior

Please give some details of what is happening. Include a [minimum complete
verifiable example] with:

- minimized playbook to reproduce the error
- the output of running ansible-lint including the command line used
- if you're getting a stack trace, also the output of
  `ansible-playbook --syntax-check playbook`

```paste below

```

[minimum complete verifiable example]: http://stackoverflow.com/help/mcve
